examples/install_vars.yaml

```diff
@@ -11,3 +11,7 @@ log_level: info
 release_image_override: ""
 
 rhcos_kernel_options: []
+
+proxy_url: ""
+cidr: ""
+
```

Comment:
is noProxy only single CIDR or can be multiple ? Also would it make sense to call out cidr as noproxy_cidr ?

Reply:
Right now only 1 network is supported hence only one CIDR is fine I think. However we can send comma seperated value if multiples are needed.
I can call it network_cidr which could be used for other purposes if needed (not restricting it to noproxy).

Reply:
Hmm, let's restrict the variable to its current use case.  On the multiple values, what do you mean by 1 network is supported? Where is this restriction coming from ?

Reply:
The automation works on a single network I mean. Reffering to https://github.com/ocp-power-automation/ocp4-upi-powervm.

Reply:
> is noProxy only single CIDR or can be multiple ? Also would it make sense to call out cidr as noproxy_cidr ?

Can I suggest to rename `cidr` to `no_proxy` (comma seperated string of domains/cidr to exclude) instead of noproxy_cidr?

Reply:
Yeah if IP is also allowed then it makes sense to use a generic var name like no_proxy.  Thanks on clarifying the single network aspect

Reply:
Done, please review again. Thanks.